src/services/MeasureService.ts

```diff
@@ -88,4 +89,34 @@ export class MeasureService implements Service<fhir4.Measure> {
 
     return createMeasurePackageBundle(measure[0]);
   }
+
+  /**
+   * result of sending a POST or GET request to:
+   * {BASE_URL}/4_0_1/Measure/$data-requirements or {BASE_URL}/4_0_1/Measure/:id/$data-requirements
+   * creates a Library with all data requirements for the specified measure
+   * requires parameters id and/or url and/or identifier, but also supports version as supplemental (optional)
```

Comment:
Should we add something about periodStart and periodEnd in this? I know this might change